### PR TITLE
Bump springboot from 2.7.9 to 2.7.11

### PR DESCRIPTION
**Pull request by Shoothzj**
## Brief changelog
Bump springboot from 2.7.9 to 2.7.11.

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [x] Update the [`CHANGES` log](https://github.com/apolloconfig/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by nobodyiam** [comment]
It looks like the snakeyml version referenced in [2.7.10](https://central.sonatype.com/artifact/org.springframework.boot/spring-boot-starter/2.7.10/dependencies) is still 1.30.

image

**Comment by Shoothzj** [comment]
@nobodyiam My apologizes, I will wait for next springboot release.

**Comment by nobodyiam** [comment]
review again

**Comment by nobodyiam** [comment]
Hello, I am a [serverless review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/). Here are my reviews of code commits in this PR.

------

### Commit 1
The patch is about upgrading the version of Spring Boot from 2.7.9 to 2.7.10. There are no potential problems identified in this patch, and it appears to be a straightforward update.

File: pom.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
 		<java.version>1.8</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<apollo-java.version>2.1.0</apollo-java.version>
-		<spring-boot.version>2.7.9</spring-boot.version>
+		<spring-boot.version>2.7.10</spring-boot.version>
```
Comment (Anilople) [inline pom.xml:67]: https://github.com/spring-projects/spring-boot/releases/tag/v2.7.11

spring boot 2.7.11 is released

Comment (Shoothzj) [reply pom.xml]: Updated

**Comment by Shoothzj** [comment]
@nobodyiam @Anilople Although this springboot version doesn't bump snakeyaml's version. But I think it's also worth to update it because it also solve other cves. Like `CVE-2023-20863` etc.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**